pkg/horizon/deployment.go

```diff
@@ -193,3 +168,91 @@ func Deployment(
 
 	return deployment, nil
 }
+
+func getEnvVars(configHash string, enabledServices map[string]string) map[string]env.Setter {
+
+	envVars := map[string]env.Setter{}
+
+	envVars["KOLLA_CONFIG_STRATEGY"] = env.SetValue("COPY_ALWAYS")
+	envVars["ENABLE_DESIGNATE"] = env.SetValue("yes")
+	envVars["ENABLE_HEAT"] = env.SetValue("yes")
+	envVars["ENABLE_IRONIC"] = env.SetValue("yes")
+	envVars["ENABLE_MANILA"] = env.SetValue("yes")
+	envVars["ENABLE_OCTAVIA"] = env.SetValue("yes")
+	envVars["ENABLE_WATCHER"] = env.SetValue(enabledServices["watcher"])
+	envVars["CONFIG_HASH"] = env.SetValue(configHash)
+
+	return envVars
+}
+
+func formatLivenessProbe() *corev1.Probe {
+
+	return &corev1.Probe{
+		TimeoutSeconds:      5,
+		PeriodSeconds:       10,
+		InitialDelaySeconds: 10,
+		ProbeHandler: corev1.ProbeHandler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: horizonDashboardURL,
+				Port: intstr.FromString(horizonContainerPortName),
+			},
+		},
+	}
+}
+
+func formatReadinesProbe() *corev1.Probe {
+
+	return &corev1.Probe{
+		TimeoutSeconds:      5,
+		PeriodSeconds:       10,
+		InitialDelaySeconds: 10,
+		ProbeHandler: corev1.ProbeHandler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: horizonContainerPortName,
```

Comment:
This is probably the problem, this should be using the `horizonDashboardURL` as well, like the others.
https://github.com/openstack-k8s-operators/horizon-operator/pull/365/files#diff-e1c0b69268828ec2f801a92017d9545c992042890e8e8b3a47c3858b73d1b860L72

Reply:
This change is a smaller subset of this change, which I probably should have done to begin with:
https://github.com/openstack-k8s-operators/horizon-operator/pull/407

We'll find out if that's the issue over on that change.